console/src/app/modules/info-row/info-row.component.ts

```diff
@@ -73,7 +73,7 @@ export class InfoRowComponent {
     if (!this.user) {
       return undefined;
     }
-    return '$typeName' in this.user ? undefined : this.user.details?.creationDate;
+    return '$typeName' in this.user ? this.user.details?.creationDate : undefined;
```

Comment:
I think the in operator is not even needed anymore:
```ts
public get changeDate() {
    return this.user?.details?.changeDate;
  }

  public get creationDate() {
    return this.user?.details?.creationDate;
  }
  ```
  Could you also remove the ? operator from this in changeDate as it's not needed.
  Please add this change a check if your use case still works.